CHANGELOG.md

```diff
@@ -4,12 +4,44 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### New and Improved
+
+* Improve performance of grants query by reducing the number of rows that need
+  to be returned. ([PR](https://github.com/hashicorp/boundary/pull/5128))
+* Add several indexes to database tables to improve performance of cascading
+  deletes/updates to session tables.
+  ([PR](https://github.com/hashicorp/boundary/pull/5128))
+* Reorder indexes on several join tables to improve performance of grants query.
+  ([PR](https://github.com/hashicorp/boundary/pull/5128))
+* Make client cache sqlite database persistent between restarts of the client
+  cache daemon. ([PR](https://github.com/hashicorp/boundary/pull/5128))
+* Improve client cache performance by adding indexes, limiting results,
+  and insuring only one refresh is running at a time for a given user and
+  resource. ([PR](https://github.com/hashicorp/boundary/pull/5128))
+
```

Comment:
It's missing https://github.com/hashicorp/boundary/pull/5101 and https://github.com/hashicorp/boundary/pull/5107

Reply:
Do you have a summary line for those? I'm not entirely sure what each one is doing. Is #5101 updating the client API to support pagination and then updating the client cache to use the pagination for targets? And then #5107 is updating the client cache to use this pagination for the remaining resources?

Reply:
I added these, let me know if that looks good.

Reply:
Looks good, thank you!